app/lib/common/utilities/address.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+
+@immutable
+class Address {
+  final String streetAddress;
+  final String city;
+  final String provinceOrState;
+  final String country;
```

Comment:
I image the String type to be replaced with enum in the future for robustness, depends if we use a location picker or something in the ATR editing screen.